python/pyspark/errors/error_classes.py

```diff
@@ -667,6 +667,11 @@
       "Only a single trigger is allowed."
     ]
   },
+  "PACKAGE_NOT_INSTALLED" : {
+    "message" : [
+      "<package_name> >= <minimum_version> must be installed; however, it was not found."
```

Comment:
@itholic, the code here fails:

```python
>>> from pyspark.sql.pandas.utils import require_minimum_pyarrow_version
>>> require_minimum_pyarrow_version()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/.../spark/python/pyspark/sql/pandas/utils.py", line 68, in require_minimum_pyarrow_version
    raise PySparkImportError(
  File "/.../spark/python/pyspark/errors/exceptions/base.py", line 46, in __init__
    self._message = self._error_reader.get_error_message(
  File "/.../spark/python/pyspark/errors/utils.py", line 39, in get_error_message
    assert set(message_parameters_from_template) == set(message_parameters), (
AssertionError: Undefined error message parameter for error class: PACKAGE_NOT_INSTALLED. Parameters: {'package_name:': 'PyArrow', 'minimum_version': '4.0.0'}
```

This breaks scheduled Python build https://github.com/apache/spark/actions/runs/7088462269/job/19291103742. I an reverting this.

Reply:
Oh,,, seems like the special character `>` somehow breaks the build while parsing the error message format. Let me made a followup PR to fix it. Thanks for addressing this!

Reply:
Oh..

Reply:
Thank you for reverting.